applications/osb-portal/src/components/workspace/NewWorkspaceToolBox.tsx

```diff
@@ -26,6 +26,22 @@ const useStyles = makeStyles((theme) => ({
   dialogButtons: {
     paddingRight: 0,
   },
+  cardHeading: {
+    textAlign: "center",
+    marginBottom: "20px",
```

Comment:
margins and paddings should come from theme.spacing or other relative unit (em) unless there is a good reason to do otherwise

Reply:
maybe the card can be directly styled on the theme too